internal/app/api/v1/scripts.go

```diff
@@ -443,7 +444,9 @@ func (s testkubeAPI) ListArtifacts() fiber.Handler {
 	return func(c *fiber.Ctx) error {
 
 		executionID := c.Params("executionID")
-
+		if s.Storage == nil || reflect.ValueOf(s.Storage).IsNil() {
```

Comment:
Reflection looks odd

Reply:
Agree. Can we find a more elegant way?

Reply:
it is either of these options
https://play.golang.org/p/QlObAWkiVdB
i could cast s.Storage.(*minio.Client) == nil

Reply:
addressed it